src/components/navigation-header/components/product-page-content/utils/__tests__/get-nav-items.test.ts

```diff
@@ -259,4 +259,35 @@ describe('getNavItems', () => {
 		]
 	`)
 	})
+
+	it('for HCP, returns documentation nav link without dropdown', () => {
```

Comment:
Added a test here to make sure the nav items are working for HCP @kendallstrautman

Reply:
💯 🙇‍♀️